modules/dns_dnscmd/dns_dnscmd_main.rb

```diff
@@ -67,6 +67,7 @@ def execute cmd, msg=nil, error_only=false
         std_err.close unless std_in.nil?
       end
       report msg, response, error_only
+      response
```

Comment:
@witlessbird: Please mind this change I need to make in order to be able to use the `execute()` I can't see any side effects, but maybe you see a better way.